sample-apps/deepedit_spleen/lib/train.py

```diff
@@ -137,6 +139,7 @@ def val_iteration_update(self):
         )
```

Comment:
https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/utils/train/basic_train.py#L206

Override this method to pass addition event names which are expecting to get fired.. in this case.. you are using events defined in IterationEvents.xyz